packages/gatsby/src/utils/page-data-helpers.ts

```diff
@@ -0,0 +1,50 @@
+import { IGatsbyPage } from "../redux/types"
+
+export interface IPageData {
+  componentChunkName: IGatsbyPage["componentChunkName"]
+  matchPath?: IGatsbyPage["matchPath"]
+  path: IGatsbyPage["path"]
+  staticQueryHashes: Array<string>
+}
+
+export function constructPageDataString(
+  {
+    componentChunkName,
+    matchPath,
+    path: pagePath,
+    staticQueryHashes,
+  }: IPageData,
+  result: string | Buffer
+): string {
+  let body = `{
+    "componentChunkName": "${componentChunkName}",
+    "path": JSON.stringify(pagePath),
```

Comment:
I've messed it up :)

```suggestion
    "path": ${JSON.stringify(pagePath)},
```

Reply:
lol, I didn't catch that either :)